tools/rosnode/src/rosnode/__init__.py

```diff
@@ -613,12 +613,17 @@ def _rosnode_cmd_info(argv):
     Implements rosnode 'info' command.
     """
     args = argv[2:]
-    parser = OptionParser(usage="usage: %prog info node1 [node2...]", prog=NAME)
+    parser = OptionParser(usage="usage: %prog info [options] node1 [node2...]",
+                          prog=NAME)
+    parser.add_option("-q", "--quiet",
+                      dest="non_verbose", default=False,
```

Comment:
Please avoid `dest`. The option name should already be clear enough to map to a variable. Also the `default` argument is not necessary for a boolean action.